src/lib/components/sidebar.svelte

```diff
@@ -85,11 +85,6 @@
 					styles="w-8 h-8 desktop:w-6 desktop:h-6 laptop:w-6 laptop:h-6 half:w-6 half:h-6 text-primary-400"
 				/></a
 			>
-			<a href="https://docs.openregistry.dev/"
-				><ReportIcon
-					styles="w-8 h-8 desktop:w-6 desktop:h-6 laptop:w-6 laptop:h-6 half:w-6 half:h-6 text-primary-400"
-				/></a
-			>
 			<a href="/search"
```

Comment:
these anchor tags should have `rel=noreferrer` & `target=_blank`

Reply:
hey! all done :)